### ISSUE TITLE
byte[] fromBase64(String hex) param incorrectly named

### ISSUE THREAD

**Issue by doridori** [defuse/password-hashing#84]
https://github.com/defuse/password-hashing/blob/master/PasswordStorage.java#L191

shows the method interface `byte[] fromBase64(String hex)` which is strange - why is the input param called `hex` (as its not hexadecimal)?

**Comment by defuse** [comment]
Yeah, that's an error. It used to be hex encoded and I forgot to change the parameter name.
